CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -1,5 +1,30 @@
 # Changelog 2.1
 
+## 2.1.1 - 2025-02-11
+
+### Added
+
+- Add documentation for the `NodeLatencyMonitor` feature. ([#6561](https://github.com/antrea-io/antrea/pull/6561), [@antoninbas])
+
+### Changed
+
+- Close connection to IPFIX collector explicitly on Stop for `FlowAggregator`. ([#6635](https://github.com/antrea-io/antrea/pull/6635), [@antoninbas])
+- Run the `IPPool` webhook handler when `SecondaryNetwork` is enabled. ([#6691](https://github.com/antrea-io/antrea/pull/6691), [@luolanzone])
+
+### Fixed
+
+- Install OpenFlow entries by PortStatus to fix an Antrea Agent failure on Windows when the OF port allocation takes longer than 5s. ([#6763](https://github.com/antrea-io/antrea/pull/6763) [#6889](https://github.com/antrea-io/antrea/pull/6889), [@wenyingd])
+- Reconcile Pods with `hostNetwork` after Antrea Agent is restarted on Windows. ([#6944](https://github.com/antrea-io/antrea/pull/6944), [@wenyingd])
+- Match `dstIP` in `ClassifierTable` to fix a potential source MAC and IP mismatched issue on Windows when `promiscuous` mode is enabled. ([#6528](https://github.com/antrea-io/antrea/pull/6528), [@XinShuYang])
+- Ensure that `promote_secondaries` is set on `IPAssigner` interfaces to avoid the automatic removal of all other IP addresses in the same subnet, which might be caused by the deletion of the primary IP address. ([#6898](https://github.com/antrea-io/antrea/pull/6898) [#6900](https://github.com/antrea-io/antrea/pull/6900), [@antoninbas])
```

Comment:
... of all other IP addresses in the same subnet, when the primary IP address is deleted